### DISCUSSION TITLE
Automations - Query Rows only allows to select a table

### DISCUSSION THREAD

**Discussion by poirazis** [Budibase/budibase#7695] [General]
The Query Rows action doesn't allow to select a query for its input, but only select a table and set filters.

Shouldn't queries also be available to select ?

**Comment by melohagan** [comment]
Hey @poirazis 

I don't believe so. It might be the name isn't completely clear, but Query Rows is for filtering table data.

If you want to use custom queries and REST queries, then you'll need to use the **External Data Connector**
![Screenshot 2022-09-13 at 15 54 19](https://user-images.githubusercontent.com/101575380/189934665-390f4464-b2f8-45f1-8a7d-1eaebc4a353b.png)
